pkg/crd/crdvalidations.go

```diff
@@ -54,6 +54,7 @@ var (
 					Description: "RequestsPerPod indicates the maximum number of concurrent requests that can be served by a specialized pod.\n This is optional. If not specified default value will be taken as 1",
 				},
 			},
+			XPreserveUnknownFields: boolPtr(true),
```

Comment:
I don't think this is needed - since `RequestsPerPod` has equivalent schema validation, did you see find any issues during tests if this is not given?

Reply:
This is so that we can add fields inside `spec` in the future and won't have to ask end users to apply the CRDs, until we figure out the schema for next version.